README.md

```diff
@@ -768,3 +769,15 @@ ErrorRuntime() error
 ```go
 school() string
 ```
+
+### Template
+
+Generate custom documents using golang's template engine.
+
+```go
+Template(template string, lines int) ([]byte, error)
+TemplateDocument(sections int) (string, error)
+TemplateMarkdown(sections int) (string, error)
+TemplateEmail(sections int) (string, error) 
```

Comment:
TemplateEmailText - lets change it to TemplateEmailText for now in the event we do an html version in the future.